agentlocal/agent_local.go

```diff
@@ -53,6 +65,17 @@ func (als *AgentLocalServer) getMetadata() agentpb.AgentServerMetadata {
 	return *als.serverMetadata
 }
 
+func (als *AgentLocalServer) getAgentsList() (res []*agentlocalpb.AgentInfo, err error) {
```

Comment:
In general, avoid named results, they have too many problems; for an extreme example: https://twitter.com/mvdan_/status/1115300377434443776
Let's say we use named results only when something fancy is done with them. For example, setting `err` with `defer`/`recover()`. When nothing fancy happens, do not use them.